resources/nl-core/nl-core-Procedure.xml

```diff
@@ -14,6 +14,7 @@
       <use value="work" />
     </telecom>
   </contact>
+  <description value="This profile covers the recording of therapeutic or diagnostic procedure undergone by the patient. The ServiceRequest profile (http://nictiz.nl/fhir/StructureDefinition/nl-core-Procedure-ServiceRequest) covers the procedure that the patient will undergo." />

```

Comment:
Missing the original description from the zib (concept)

Reply:
#66 - The whole zib original description is not fitting her because of the split in Procedure / ServiceRequest. Do you have any comments on the current description?